src/django/api/serializers/activity_log.py

```diff
@@ -0,0 +1,68 @@
+from rest_framework.serializers import (
+    ModelSerializer,
+    SerializerMethodField,
+    EmailField,
+    DateTimeField,
+)
+
+from ..models import User, Submission
+
+
+class ActivityDraftedSerializer(ModelSerializer):
+    user = EmailField(source='created_by.email')
+    action = SerializerMethodField()
+    time = DateTimeField(source='created_at')
+
+    def get_action(self, value):
+        return 'Drafted'
+
+    class Meta:
+        model = Submission
+        fields = ['user', 'action', 'time', 'notes']
+        optional_fields = ['annotations_count']
+
+
+class ActivitySubmittedSerializer(ActivityDraftedSerializer):
+    def to_representation(self, instance):
+        obj = super().to_representation(instance)
+        submitter = User.objects.get(pk=instance.submitted_by.pk)
+        obj['action'] = 'Submitted'
+        obj['time'] = instance.submitted_at
+        obj['user'] = submitter.email
+        return obj
+
+
+class ActivityReviewStartedSerializer(ActivityDraftedSerializer):
+    def to_representation(self, instance):
+        obj = super().to_representation(instance)
+        review = instance.review
+        reviewer = User.objects.get(pk=review.reviewed_by.pk)
```

Comment:
Seeing the contributor as the user for Review activities: http://localhost:8181/api/boundaries/4/

![image](https://user-images.githubusercontent.com/1430060/196277335-38c66142-f104-497a-af13-9371b2d505d8.png)

It should be the validator.